### ISSUE TITLE
(Mob Idea) Griffins

### ISSUE THREAD

**Issue by OakTree1203** [AlexModGuy/Ice-and-Fire-Suggestions#59]
Griffin Ideas

1. Size: 
Stage One: Pig
Stage Two: Cow
Stage 3: Horse
Stage 4: A Horse And A Half/A Llama
Stage 5: Two Horses

2. Appearance: What you've posted on the front page under the "Future Creatures" spoiler is perfect!
3. Where It's Found: In Hills or Extreme Hills. There could be mating pairs that guard a nest,
which may or may not have eggs.

4. Behaviors: As mentioned earlier, they could spawn guarding nests, 
which might contain eggs.
Extremely hostile, they will kill anything they deem a threat due to it:
a) Coming too close to the nest.
b) Attacking one of the two griffins.
c) Being a predator. (Dragon vs. Griffins anyone?)
If either of the pair stray to far from the nest for whatever reason, they will both attempt to 
return to the nest.
(To make sure the eggs are safe.)
The pair will circle around the nest, 
watching it from the air.
Their aggro range is bigger than any vanilla mob's. 
(Except for the bosses.)
Starting at stage three, dragons will attack any griffins 
that are any lower stage than their own, and griffins will do the same to dragons.
If a newly hatched griffin (Stage one or two) sees a dragon or a player it will attempt to run away.

5. Eggs/Development: After a while in the wild, the eggs will hatch. 
(About 2-3 in game days after they are initially loaded in.)
You can find anywhere from zero to five eggs in a nest.
Not all eggs will hatch. 
For simplicity's sake, griffins 
of all stages could appear similar except for the size.
Similar to the dragons, there could be five griffin stages. 
At stage three, they are able to fly.
You will not require a saddle.

6. Taming: To tame one of these beautiful creatures, you will need to steal an egg.
Try and get as many as you can, as nests are not very common and not every egg will hatch.
Eggs can withstand extreme amounts of heat and cold, but I wouldn't recommend putting them in lava.
A young griffin is very vulnerable, so keep it somewhere very safe when you aren't directly supervising it.
Egg: Five Hearts
Stage One Health: Ten Hearts
Stage Two Health: Twenty Hearts
Stage Three Health: Twenty Five Hearts
Stage Four Health: Thirty Five Hearts
Stage Five Health: Fifty Hearts
The amounts of health could be the same in the wild.
To pick up an egg/place it in your inventory, simply right click on it.
The tricky part is actually getting the egg(s) and getting out alive.
If you go too close to the nest, both parents will dive down on you.

6.5. Strategies:
1. Kill both parents will a long range weapon and steal the eggs.
2. Try ender pearling towards the nest, grab what you can, and teleport out.
Keep in mind the guardians of the nest will be on top of you in seconds.
3. Get towards that nest as fast as possible, swipe some eggs, 
and fight like your
life depends on it, because it does.
4. If you wanted to be really cheesy, you could just turn on keep inventory, grab
all the eggs, let yourself die, and respawn at your bed.

7. Conclusion: I know the Griffin may not be the next mob, or even the one after
that, but these are just suggestions.

**Comment by LordDecay26** [comment]
there are already very fleshed out plans for a hippogriff. a griffon, nor suggestions for a griffin or a hippogriff are not necessary. hippogriff's will spawn in Desert hills, Birch forest mountains, Taiga mountains, ice mountains, extreme hills, Savannah mountains, and roofed forest mountains. in order to tame a hippogriff one must feed it a lot of rabbit meat over and over again until it is tamed, hippogriffs can be ridden, chested, and armored. they also follow you like a wolf. 

though, i might consider this as an add-on =) because these griffons sound neato

**Comment by OakTree1203** [comment]
Thanks for telling me this, I had absolutely no idea. Although, Desert Hills sounds like a bit of an odd area to live in for such a large predator. Yes, there's a fair amount of rabbits, but I'd assume one animal would have to eat quite a few a day.

**Comment by LordDecay26** [comment]
its not really that large, only about the size of a horse. mind you, donkeys spawn in desert hills as well as the occasional llama, both of which will be natural prey to the hippogriff.

**Comment by OakTree1203** [comment]
Yeah, to my knowledge/experience with WoW, (grown) Griffins and Hippogryphs are large enough for someone to very comfortable ride on, with the Griffins larger than the Hippogryphs. (Lions are bigger than horses, and eagles are larger than ravens (what I believe the bird portion of a hippogryph is in WoW))

**Comment by LordDecay26** [comment]
yeah. i would like the idea of them adding in both... maybe even making griffins smaller than hippogriffs and base the back end of them from the ocelot. ;) it could be an epic dynamic. a small griffin could act as a household pet rather than a war mount. very cute :3

**Comment by OakTree1203** [comment]
This is nice, but it's been said that there's no plans for a Gryffin, and
this seems a little simplistic to be a part of a mod with such complex
mobs. Personally, I'd like it, but I don't think it's going to happen.
